app/pages/system/inventory/sled/SledInstancesTab.tsx

```diff
@@ -43,38 +44,49 @@ const makeActions = (): MenuAction[] => []
 
 export function SledInstancesTab() {
   const { sledId } = useSledParams()
-  const { Table, Column } = useQueryTable(
+  const { Table } = useQueryTable(
     'sledInstanceList',
     { path: { sledId }, query: { limit: 25 } },
     { placeholderData: (x) => x }
   )
+  const colHelper = createColumnHelper<SledInstance>()
+  const staticCols = [
+    colHelper.accessor((i) => pick(i, 'name', 'siloName', 'projectName'), {
+      header: 'name',
+      id: 'name',
+      cell: (info) => {
+        const value = info.getValue()
+        return (
+          <div className="space-y-0.5">
+            <div className="text-quaternary">{`${value.siloName} / ${value.projectName}`}</div>
+            <div className="text-default">{value.name}</div>
+          </div>
+        )
+      },
+    }),
+    colHelper.accessor('state', {
+      header: 'status',
+      id: 'status',
+      cell: (info) => <InstanceStatusBadge key="run-state" status={info.getValue()} />,
+    }),
+    colHelper.accessor((i) => pick(i, 'memory', 'ncpus'), {
+      header: 'specs',
+      id: 'specs',
+      cell: (info) => <InstanceResourceCell value={info.getValue()} />,
+    }),
+    colHelper.accessor('timeCreated', {
+      header: 'created',
+      id: 'created',
+      cell: (info) => <DateCell value={info.getValue()} />,
+    }),
+    colHelper.accessor('timeModified', {
+      header: 'modified',
+      id: 'modified',
+      cell: (info) => <DateCell value={info.getValue()} />,
+    }),
```

Comment:
The `id`s here can all be removed — if the accessor is a string key, RT will use that as the ID, and if it's a function it will use `header` as the ID.

https://tanstack.com/table/latest/docs/guide/column-defs#unique-column-ids